app/lib/app/SecvisogramPage/View/FormEditor/editors/GenericEditor.js

```diff
@@ -255,6 +276,7 @@ export default function Editor({
           value={value || ''}
           property={property}
           disabled={disabled}
+          fillFunction={fillFunction}
```

Comment:
It's currently affecting only fields of type `STRING` and is set above according to the more fine grained uiTypes. In case a `fillFunction` is provided for other types, the additional checks (for uiType etc.) will need to be added and a function can be passed to the corresponding Editor instances.

Then in the specific editors it's just passed through to the `Attribute` class which actually takes care of adding the button to call the function.

Actually, the case for `STRING_GENERATE_PRODUCT_ID` can also be simplified. Let me tackle that in another branch, where I will take care also of the reset of the ID counter.